scripts/manifest.js

```diff
@@ -44,10 +45,22 @@ const overrides = new Map([
 
 const icons = manifest.map(icon => {
 	const id = typeof icon === 'string' ? icon : icon.symbol;
+	const fileName = `${id.toLowerCase()}.svg`;
+	const svgPath = path.join(__dirname, '..', 'svg', 'color', fileName);
+	let color;
+	if (fs.existsSync(svgPath)) {
+		const svg = fs.readFileSync(svgPath, 'utf8');
+		const colors = getColors(svg);
+		const hex = colors.fills.map(color => color.hex().toUpperCase())[0];
+		color = hex;
+	} else {
+		color = 'not_found';
```

Comment:
I think `undefined` would be better than `"not_found"` if there is no colour.

We don't want people blindly setting colours to `"not_found"`. If the value is undefined it's a lot simpler to check for and won't cause as many issues if it's used blindly.